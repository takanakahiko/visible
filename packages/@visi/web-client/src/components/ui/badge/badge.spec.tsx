import { render } from '@testing-library/react';
import React from 'react';

import { Badge } from '.';

describe('Badge', () => {
  it('matches snapshot', () => {
    const { container } = render(<Badge variant="red">test</Badge>);
    expect(container.firstChild).toMatchSnapshot();
  });
});
