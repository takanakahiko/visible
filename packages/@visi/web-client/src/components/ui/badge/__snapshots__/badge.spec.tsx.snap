// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Badge matches snapshot 1`] = `
<div
  class="inline-flex items-center"
>
  <div
    class="w-2 h-2 m-1 mr-2 rounded-full bg-red-600"
    role="presentation"
  />
  <span
    class="text-sm text-gray-800 leading-none"
  >
    test
  </span>
</div>
`;
